packages/ui-components/src/components/illustration/readme.md

```diff
@@ -53,16 +53,10 @@ export const SvgIconExample: React.FC = () => (
 
 ## CSS Custom Properties
 
-| Name                            | Description                                        |
-| ------------------------------- | -------------------------------------------------- |
-| `--illustration-color`          | Base color used in the illustration                |
-| `--illustration-color-dark-35`  | 35% darker version of the illustration base color  |
-| `--illustration-color-dark-55`  | 55% darker version of the illustration base color  |
-| `--illustration-color-light-80` | 80% lighter version of the illustration base color |
-| `--illustration-color-light-90` | 90% lighter version of the illustration base color |
-| `--illustration-height`         | Height of illustration                             |
-| `--illustration-success-color`  | Success color used in the illustration             |
-| `--illustration-width`          | Width of illustration                              |
+| Name                    | Description            |
+| ----------------------- | ---------------------- |
+| `--illustration-height` | Height of illustration |
+| `--illustration-width`  | Width of illustration  |
```

Comment:
please confirm why some css custom properties are not here

Reply:
That's odd, I'm going to check out :+1: